### PR TITLE
LPD-42499 Technical Task | Replace text for Page Conflict message on Check Conflict screen

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: https://liferay.atlassian.net/browse/LPD-42499, https://liferay.atlassian.net/browse/LPD-42498

I've researched how to deploy, and looks like it is not possible to deploy  the liferay learn resources, so right now the link is not redirecting to any place https://liferay.slack.com/archives/CNBG06JS3/p1704951376033559?thread_ts=1704836440.687919&cid=CNBG06JS3

I attached a video where I used the other liferay learn link that is available, just for testing purposes: https://github.com/user-attachments/assets/011da390-7c6e-4407-89c3-f846b4bc2113

One thing that I'm not sure if it's used in the right way is the liferay learn resource, because I'm not using the message for anything.  A possibility was to add the message just as 'Documentation', but I thought it could be way too different of the pattern used, and also the word documentation was in lower case in the design.

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:stop

**Comment by gislayne-vitorino** [comment]
ci:test:Sf

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -6262,6 +6262,7 @@ document-types=Document Types
 document-updated-email=Document Updated Email
 document-util=Document Util
 document-x-of-x=Document {0} of {1}
+documentation=documentation
```
Comment (david-truong) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:6265]: @gislayne-vitorino doesn't the value need to be capitalized?

Comment (gislayne-vitorino) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:6265]: @david-truong Yeah, that's what I thought, but I wasn't sure how I could do it differently and still match the design

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:stop

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by david-truong** [comment]
@gislayne-vitorino did you update the logic on when the text will appear?  Also the new text is a bit off.  Can you start a thread on slack and we can discuss what the new text should be.

**Comment by gislayne-vitorino** [comment]
@david-truong no, the task was just for a replacement so I didn't update it, and by saying that I just noticed that I need to update the old tests as well. I will update them after we decide on the text, I will open a thread on our channel

**Pull request closed without merging**